components/brave_extension/extension/brave_extension/background/api/cosmeticFilterAPI.ts

```diff
@@ -22,7 +38,37 @@ export const removeSiteFilter = (origin: string) => {
   })
 }
 
-export const applySiteFilters = (tabId: number, hostname: string) => {
+export const applyAdblockCosmeticFilters = (tabId: number, hostname: string) => {
```

Comment:
this is really not the right way to do this and the same thing applies here https://github.com/brave/brave-core/pull/3303/files#diff-3493bfe26c7890d60718508a5969519aR84
we should be using the declarative content api to add and remove rules when the settings change https://developer.chrome.com/extensions/declarativeContent